crates/lib3h/src/transport/websocket/streams.rs

```diff
@@ -388,7 +388,7 @@ impl<T: Read + Write + std::fmt::Debug> StreamManager<T> {
                         }
                         Err(e.into())
                     }
-                    Err(tungstenite::error::Error::ConnectionClosed(_)) => {
+                    Err(tungstenite::error::Error::ConnectionClosed) => {
                         // close event will be published
                         Ok(())
```

Comment:
Should we have a log that we encountered an error here ?

Reply:
Done